backend/src/oe/api/routes.ts

```diff
@@ -212,6 +212,20 @@ class OpEnergyRoutes {
     }
     logger.info( `${UUID}: PROFILE: end: $getBlockByHash`);
   }
+
+  private async $getBlockSpanList(req: Request, res: Response) {
+    const UUID = await opEnergyApiService.$generateRandomHash();
+    try {
+      logger.info( `${UUID}: PROFILE: start: $getBlockSpanList`);
+      const { startBlockHeight, span, numberOfSpan } = req.params;
```

Comment:
I am worried about an absence of input validation here: either of `startBlockHeight, span, numberOfSpan` can have string value and such variables will have `NaN` values. If any of those variables will be negative, then they will be passed as negative numbers further to opEnergyApiService.$getBlockSpanList() call.

Reply:
done